### PR TITLE
updates roles read-only

### PR DESCRIPTION
**Pull request by RodrigoDLopez**
### Description

The ACS default rules for read-only accounts (Read-Only Admin - Default and Read-Only User - Default) do not allow accounts created with these rules to execute the `quotaStatement` and `quotaBalance` commands; for this reason, quota consumption history is not displayed for user-accounts created with these roles.

This PR introduces an update to the default rules to allow read-only accounts to cal these APIs

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

### CODE REVIEW

**Review by rohityadavcloud — APPROVED** [review]
LGTM, didn't test it

File: engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql [hunk 1]
```diff
@@ -998,6 +998,38 @@ BEGIN
...
+
+INSERT  INTO `cloud`.`role_permissions` (uuid, role_id, rule, permission, sort_order)
+SELECT  UUID(), role_id, 'quotaStatement', 'ALLOW', MAX(sort_order)-1
+FROM    `cloud`.`role_permissions` RP
+WHERE   role_id = (SELECT id FROM `cloud`.`roles` WHERE name = 'Read-Only User - Default');
+
+INSERT  INTO `cloud`.`role_permissions` (uuid, role_id, rule, permission, sort_order)
+SELECT  UUID(), role_id, 'quotaBalance', 'ALLOW', MAX(sort_order)-2
+FROM    `cloud`.`role_permissions` RP
+WHERE   role_id = (SELECT id FROM `cloud`.`roles` WHERE name = 'Read-Only User - Default');
```
Comment (DaanHoogland) [inline engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql:1031]: executing this sec i get:
```
Error occurred during SQL script execution

Reason:
SQL Error [1140] [42000]: (conn=130) In aggregated query without GROUP BY, expression #2 of SELECT list contains nonaggregated column 'cloud.RP.role_id'; this is incompatible with sql_mode=only_full_group_by
```

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

File: engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql (hunk 1 above)
Comment (DaanHoogland) [reply engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql:1031]: tried again in a new environment and it worked.

**Review by DaanHoogland — APPROVED** [review]
tested manually and in a regression test suite.

**Pull request merged**